client/client.go

```diff
@@ -42,6 +42,7 @@ type Config struct {
 	Headers          http.Header
 	TLS              TLSConfig
 	DialContext      func(ctx context.Context, network, addr string) (net.Conn, error)
+	Verbose          bool
```

Comment:
should also be exposed on the CLI i thinnk but this an old PR so will just merge 👍